io.missioncenter.MissionCenter.json

```diff
@@ -0,0 +1,232 @@
+{
+  "app-id": "io.missioncenter.MissionCenter",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "command": "missioncenter",
+  "build-options": {
+    "no-debuginfo": true
+  },
```

Comment:
That one is weird. Does it fix some build issue?

Reply:
Yes, it prevents flatpak-builder from extracting debug info from the Rust installation and breaking it in the process